lib/build.api.js

```diff
@@ -192,7 +192,8 @@ class Build {
             switch (link.conflict) {
               case 'overwrite':
                 this.aquifer.console.log('Destination exists. Conflict set to overwrite. \nOverwriting: ' + link.dest, 'status');
-                del.sync(link.dest);
+                options.force = true;
```

Comment:
@ccjjmartin This looks like it is altering the options that are passed into the command and therefore may contain options unrealted to `del.sync()`. Should we define a new variable here instead?